doc/source/whatsnew/v0.17.1.txt

```diff
@@ -50,6 +50,8 @@ Performance Improvements
 
 .. _whatsnew_0171.bug_fixes:
 
+- Performance bug in ``Categorical._repr_categories`` was rendering string before chopping them for display (:issue: `11305`)
```

Comment:
move to Performance section

say ``Performance issue in rendering a large number of categories when printing a Categorical or Series of category dtype``